src/components/NavigationBar/index.js

```diff
@@ -249,31 +250,41 @@ class NavigationBar extends Component {
                   />
                 ) : null}
                 {accessToken && (
-                  <StyledIconButton padding={'0.55rem'}>
-                    <div data-tip="custom" data-for={'right-menu'}>
-                      <FlexContainer>
-                        <Popper
-                          id={'right-menu'}
-                          place="bottom"
-                          effect="solid"
-                          delayHide={200}
-                          type={'light'}
-                          offset={{ top: -3 }}
-                        >
-                          <Paper>
-                            <Logged />
-                          </Paper>
-                        </Popper>
-                        <CircleImage
-                          name="User Avatar"
-                          src={userAvatar}
-                          size="32"
-                        />
-                        <UserDetail>{!userName ? email : userName}</UserDetail>
-                        <ExpandMore />
-                      </FlexContainer>
-                    </div>
-                  </StyledIconButton>
+                  <div>
```

Comment:
Is this extra nesting required?

Reply:
It should be in a single wrapper.

Reply:
Removing that it says:
Syntax error: Adjacent JSX elements must be wrapped in an enclosing tag. Did you want a JSX fragment <>...</>? (281:21)